### PR TITLE
Opn 711 new agreement type

### PR DESCRIPTION
**Pull request by HoussamBedja**

### CODE REVIEW

File: ckanext/canada/tables/contracts.yaml [hunk 1]
```diff
@@ -660,15 +660,27 @@ iii. Quand un contrat porte sur plus d’un article économique, il est recomman
         en: CFTA / CCFTA / CKFTA
         fr: ALEC / ALECC / ALECCS
       D:
-        lookup: [CETA, CFTA]
+        conditional_lookup:
+        - column: contract_date
+          less_than: '2021-01-01'
+          lookup: [CETA, TCA, CFTA]
+        - lookup: [CETA, CFTA]
```
Comment (wardi) [inline ckanext/canada/tables/contracts.yaml:667]: I think this is inverted. TCA is an agreement that applies after 2021-01-01, right?

File: ckanext/canada/tables/contracts.yaml [hunk 2]
```diff
@@ -660,15 +660,27 @@ iii. Quand un contrat porte sur plus d’un article économique, il est recomman
...
         en: CETA / CFTA
```
Comment (wardi) [inline ckanext/canada/tables/contracts.yaml:668]: add the new agreement to the list that will be shown in the drop-down here too

**Comment by wardi** [comment]
You need to add TCA to the `choices_lookup` below as well

File: ckanext/canada/tables/contracts.yaml (hunk 1 above)
Comment (HoussamBedja) [reply ckanext/canada/tables/contracts.yaml]: You're right. I just swapped them.

**Comment by wardi** [comment]
Looks good to me. Is there a reason the PR is still marked as a draft?

**Comment by HoussamBedja** [comment]
> Looks good to me. Is there a reason the PR is still marked as a draft?

Business team wants to test in on staging first + they're not sure yet when the client wants this live.

**Pull request merged**